commands.js

```diff
@@ -1044,16 +1044,48 @@ exports.commands = {
 			move = Commands.randommove.call(this, "2,viable", self, dummyRoom);
 		}
 
+		//Code for generating a random name based on types generated
+		//Start by accessing the array from the capnamelist.js file in the data folder.
+		var foundName = false;
+		var t1Name = '';
+		var type1array = Data.Capnames[type[0]];
+		var type2array = Data.Capnames[type[1]];
+		while(!foundName){
+            t1Name = Tools.sample(type1array)[0]
+            if(t1Name.charAt(0) === '-'){ //Case Suffix, can't be first part of name. remove and reroll
+                Tools.removeElm(type1array, t1Name);
+            }else if(t1Name.charAt(0) === '+'){ //Case Prefix, remove '+' char from string and use
```

Comment:
and example of the styling issues, this should be `} else if (t1Name.charAt(0) === '+') {`